common/flatpak-dir.c

```diff
@@ -7757,7 +7757,7 @@ flatpak_dir_find_local_ref (FlatpakDir   *self,
     }
 
   g_set_error (error, G_IO_ERROR, G_IO_ERROR_NOT_FOUND,
-               _("Can't find %s%s%s%s%s in remote %s"), name,
+               _("Can't find %s%s%s%s%s in local repository"), name,
```

Comment:
This should probably be squashed with the previous commit, but in any event you'll have to remove the last `remote` argument from the list.